.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: Ubuntu Build Test
+
+on:
+    pull_request: {}
+    push: {}
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.16
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Build Test
+      env:
+        GITHUB_ACTIONS: true
```

Comment:
`GITHUB_ACTIONS` is provided automatically, right? if not we can keep this here - otherwise it doesn't seem to add value

Reply:
fixed